herbert_core/herbert_init.m

```diff
@@ -25,7 +25,7 @@
 % end
 % Add paths
 % ---------
-addpath(herbert_path);  % MUST have herbert_path so that herbert_init, herbert_off included
+addpath(herbert_path);  % MUST have herbert_path so that herbert_init, included
```

Comment:
The source of 

Function herbert_off.m not found in /home/jacob/PACE/HorBack/herbert_core
Clearing rootpath and subdirectories from Matlab path in any case

is at L.106 in this file below.